frontend/src/pages/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -1,16 +1,63 @@
-import { Button } from '~/libs/components/components.js';
+import { Button, Input, Link } from '~/libs/components/components.js';
+import { AppRoute } from '~/libs/enums/enums.js';
+import { useAppForm, useCallback } from '~/libs/hooks/hooks.js';
+import {
+  type UserSignInRequestDto,
+  userSignInValidationSchema,
+} from '~/packages/users/users.js';
+
+import { DEFAULT_SIGN_IN_PAYLOAD } from './libs/constants.js';
+import styles from './styles.module.scss';
 
 type Properties = {
-  onSubmit: () => void;
+  onSubmit: (payload: UserSignInRequestDto) => void;
 };
 
-const SignInForm: React.FC<Properties> = () => (
-  <>
-    <h1>Sign In</h1>
-    <form>
-      <Button label="Sign in" />
-    </form>
-  </>
-);
+const SignInForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
+  const { control, errors, handleSubmit } = useAppForm<UserSignInRequestDto>({
+    defaultValues: DEFAULT_SIGN_IN_PAYLOAD,
+    validationSchema: userSignInValidationSchema,
+  });
+
+  const handleFormSubmit = useCallback(
+    (event_: React.BaseSyntheticEvent): void => {
+      void handleSubmit(onSubmit)(event_);
+    },
+    [handleSubmit, onSubmit],
+  );
+
+  return (
+    <>
+      <div className={styles.headerWrapper}>
+        <h1 className="heading-2">Log In</h1>
+        <h3 className="body-2">
+          No account?&nbsp;
+          <Link to={AppRoute.SIGN_UP}>Sign Up</Link>
+        </h3>
+      </div>
+      <form onSubmit={handleFormSubmit} className={styles.formWrapper}>
+        <Input
+          type="email"
+          label="Email"
+          placeholder="Enter your email"
+          name="email"
+          control={control}
+          errors={errors}
+        />
+        <Input
+          type="password"
+          label="Password"
+          placeholder="Enter your password"
+          name="password"
+          control={control}
+          errors={errors}
+        />
+        <div className={styles.btnWrapper}>
+          <Button type="submit" label="Log In" isSmall={true} />
```

Comment:
Boolean props with default fallback to `false` value could be passed this way, but not sure that it's a good idea to pass styling with boolean prop - won't it be better to just provided custom class to change default button sizing?
```suggestion
          <Button type="submit" label="Log In" isSmall />
```

Reply:
Done